apple/internal/ios_rules.bzl

```
@@ -1489,10 +1491,12 @@ def _ios_dynamic_framework_impl(ctx):
     )
 
     signed_frameworks = []
-    if getattr(ctx.file, "provisioning_profile", None):
-        signed_frameworks = [
-            bundle_name + rule_descriptor.bundle_extension,
-        ]
+    if codesigning_support.should_sign_bundles(
+        provisioning_profile = ctx.file.provisioning_profile,
```

The move away from `getattr(ctx.file, "provisioning_profile", None)` is acceptable here, correct?
```suggestion
        provisioning_profile = getattr(ctx.file, "provisioning_profile", None),
```

---

I'm not sure why it wasn't a direct reference to begin with. Was it ever optional/not there?

---

No clue. At least it's unified and not a smattering of getattr and direct access. I like it!